build/common/installer/scripts/tomlparser.rb

```diff
@@ -26,7 +26,7 @@
 @logEnableMultiline = "false"
 @stacktraceLanguages = "go,java,python,dotnet"
 @logEnableKubernetesMetadata = false
-@logKubernetesMetadataIncludeFields = "podLabels,podAnnotations,podUid,image"
+@logKubernetesMetadataIncludeFields = "podlabels,podannotations,poduid,image"
```

Comment:
we should give config options for customer to use all these fields also "image, imageID, imageTag, repository".  I think, renaming the field "repository" to "imageRepo" might be use for customer to understand than repository.